consts/consts.go

```diff
@@ -777,3 +817,19 @@ var ES_SRC_PARENTS_FOR_CHAPTER_POSITION_INDEX = map[string]PositionIndexType{
 	SRC_SHAMATI:               LETTER_IF_HEBREW,
 	SRC_RABASH_ASSORTED_NOTES: ALWAYS_NUMBER,
 }
+
+var ES_GRAMMAR_DIVT_TYPE_TO_SOURCE_TYPES = map[string][]int64{
+	VAR_DIV_ARTICLE: []int64{7},
+	VAR_DIV_CHAPTER: []int64{6, 7, 9},
+	VAR_DIV_VOLUME:  []int64{3},
+	VAR_DIV_PART:    []int64{4},
+	VAR_DIV_NUMBER:  []int64{6, 7, 9},
+}
+
+var NOT_TO_INCLUDE_IN_SOURCE_BY_POSITION = []string{
+	SRC_LETTERS_RABASH, SRC_ARTICLES_RABASH, SRC_ARTICLES_BAAL_SULAM, // Children 'position' value of these sources are not according to their actual chapter
+}
+
+var SOURCE_PARENTS_NOT_TO_INCLUDE_IN_VARIABLE_VALUES = []string{
```

Comment:
comment added:

// We avoid adding source names from Rabash Assorted notes because many of them are similar to concepts or topics and less known as names of Rabash sources.
// Also we avoid adding names of article summaries and campus material to avoid confusion with the original sources.